src/tests/mocks/bors.rs

```diff
@@ -294,6 +299,15 @@ impl BorsTester {
     }
 
     pub async fn push_to_pull_request(&mut self, pr_number: u64) -> anyhow::Result<()> {
+        let repo = self.default_repo();
+        let mut repo = repo.lock();
+        let count = PR_PUSH_SHA_COUNTER.fetch_add(1, SeqCst);
+
+        if let Some(pr) = repo.pull_requests.get_mut(&pr_number) {
```

Comment:
Let's crash here if the PR does not exist, the test should make sure that the PR has been initialized.